lib/src/widgets/utils/ds_header.widget.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
```

Comment:
In this widget things seem pretty obvious. In these cases are we going to suppress the documentation, or would it be nice to have some documentation in the widgets by default?

Reply:
I think that it would be nice to always have some kind of doc comments, especially if we plan to export some doc pages using ```dart doc``` in the future.